internal/cmd/connect/command_custom_plugin_update.go

```diff
@@ -0,0 +1,48 @@
+package connect
+
+import (
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
+	"github.com/spf13/cobra"
+)
+
+func (c *customPluginCommand) newUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "update <id>",
+		Short:       "Update a custom connector plugin configuration.",
+		Args:        cobra.ExactArgs(1),
+		RunE:        c.update,
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+	}
+	cmd.Flags().String("name", "", "name of plugin")
+	cmd.Flags().String("description", "", "description of plugin")
+	cmd.Flags().String("documentation-link", "", "document link of plugin")
+	cmd.Flags().String("sensitive-properties", "", "sensitive properties config of custom plugin")
+
+	pcmd.AddContextFlag(cmd, c.CLICommand)
```

Comment:
Please look at other commands for examples, but we could separate the flags with some whitespace here.
```suggestion

	cmd.Flags().String("name", "", "name of plugin")
	cmd.Flags().String("description", "", "description of plugin")
	cmd.Flags().String("documentation-link", "", "document link of plugin")
	cmd.Flags().String("sensitive-properties", "", "sensitive properties config of custom plugin")

	pcmd.AddContextFlag(cmd, c.CLICommand)
	
```